addons/mail/static/src/core/web/chatter.js

```diff
@@ -57,7 +62,7 @@ export class Chatter extends Component {
         "hasActivities?",
         "hasFollowers?",
         "hasMessageList?",
-        "hasMessageListScrollAdjust?",
+        "isChatterSidebar?",
```

Comment:
`isChatterSidebar` is indeed an improvement over `hasMessageListScrollAdjust`. I think we can use `Aside` instead of `Sidebar`, so terminology is consistent (`.o-mail-Form-chatter.o-aside`) and slightly different from discuss sidebar for example.

So `isChatterSidebar` => `isChatterAside`